src/rviz/default_plugin/path_display.cpp

```diff
@@ -499,11 +499,11 @@ void PathDisplay::processMessage( const nav_msgs::Path::ConstPtr& msg )
         const geometry_msgs::Point& pos = msg->poses[ i ].pose.position;
         Ogre::Vector3 xpos = transform * Ogre::Vector3( pos.x, pos.y, pos.z );
         axes_vect[i]->setPosition(xpos);
-        Ogre::Quaternion orientation(msg->poses[ i ].pose.orientation.w,
```

Comment:
Please use `const geometry_msgs::Quaternion& quat = msg->poses[ i ].pose.orientation` as a shortcut here.
I know, it wasn't used before, but if we touch this code anyway ...